pkg/build/common/util.go

```diff
@@ -0,0 +1,99 @@
+package common
```

Comment:
Packages `common` and files `util` are frowned upon in Go. Let's get rid of the `pkg/build/golang` nesting, so that it's at parity with `pkg/runner` -- wanted to do this for v0.5 but didn't get around to do the busywork. Then all builders will be in a flat package, and we can name this file `registries.go`, which is what these functions are about. I'll file a PR for the shifting around.